src/test/java/HW20.java

```diff
@@ -0,0 +1,85 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HW20 extends BaseTest {
+
+    @Test
+    public void DeletePlaylist() throws InterruptedException {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

_For example_ `public void deletePlaylist()`

and

Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.


![image](https://user-images.githubusercontent.com/104761482/212018462-cdf43c32-6a70-41f0-aa7a-349f916cf931.png)